shell/platform/darwin/ios/framework/Headers/FlutterPlatformViews.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_FLUTTERPLATFORMVIEWS_H_
+#define FLUTTER_FLUTTERPLATFORMVIEWS_H_
+
+#import <CoreMedia/CoreMedia.h>
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#include "FlutterCodecs.h"
+#include "FlutterMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ An instance of a `UIView` created by a `FlutterPlatformViewFactory`.
```

Comment:
I've reformatted the documentation in #6447 - the format being used previously was inconsistent and didn't play with the doc generator very well.  This one should have `*` all the way down each line.